src/patternfly/components/Timestamp/timestamp.scss

```diff
@@ -0,0 +1,43 @@
+.pf-c-timestamp {
+  // Timestamp text
+  --pf-c-timestamp__text--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-timestamp__text--Color: var(--pf-global--Color--200);
+
+  // Help text variables for the timestamp
+  --pf-c-timestamp--m-help-text--TextDecorationLine: underline;
+  --pf-c-timestamp--m-help-text--TextDecorationStyle: dashed;
+  --pf-c-timestamp--m-help-text--TextDecorationThickness: var(--pf-global--BorderWidth--sm);
+  --pf-c-timestamp--m-help-text--TextUnderlineOffset: #{pf-size-prem(4px)};
+  --pf-c-timestamp--m-help-text--TextDecorationColor: var(--pf-global--BorderColor--200);
+  --pf-c-timestamp--m-help-text--hover--Color: var(--pf-global--Color--100);
+  --pf-c-timestamp--m-help-text--focus--Color: var(--pf-global--Color--100);
+  --pf-c-timestamp--m-help-text--hover--TextDecorationColor: var(--pf-global--Color--100);
+  --pf-c-timestamp--m-help-text--focus--TextDecorationColor: var(--pf-global--Color--100);
+
+  .pf-c-timestamp__text {
+    font-size: var(--pf-c-timestamp__text--FontSize);
+    color: var(--pf-c-timestamp__text--Color);
+  }
+
+  &.pf-m-help-text {
+    cursor: pointer;
+
+    .pf-c-timestamp__text {
+      text-decoration-line: var(--pf-c-timestamp--m-help-text--TextDecorationLine);
+      text-decoration-style: var(--pf-c-timestamp--m-help-text--TextDecorationStyle);
+      text-decoration-thickness: var(--pf-c-timestamp--m-help-text--TextDecorationThickness);
+      text-underline-offset: var(--pf-c-timestamp--m-help-text--TextUnderlineOffset);
+      text-decoration-color: var(--pf-c-timestamp--m-help-text--TextDecorationColor);
+    }
```

Comment:
Looks like these styles need to apply to `pf-c-timestamp` rather than `pf-c-timestamp__text`.